src/style.css

```diff
@@ -1,3 +1,69 @@
-body {
+body,
+html {
+  padding: 0;
   margin: 0;
+  box-sizing: border-box;
+  font-family: Roboto, sans-serif;
```

Comment:
You've done well by specifying a specific font for your webpage. However, it's always a good idea to specify a fallback font in case the main one doesn't work. This is done by specifying a generic font family after the specific one. For instance, 'Roboto', sans-serif. This is a requirement from the checklist: 'Remember to use fallback fonts - alternative font-family in case the main one doesn't work'